e2e/infrastructure/TransactionHttp.spec.ts

```diff
@@ -895,7 +897,7 @@ describe('TransactionHttp', () => {
                 Deadline.create(),
                 NetworkCurrencyLocal.createAbsolute(10),
```

Comment:
We should delegate the creation of a currency mosaic to the helper so it can resolve if it's running locally or tesnet (different currency name)

Upgrade all secret lock transactions to use the new helper.

Reply:
yep. thought about it already. will do it in other PR